### ISSUE TITLE
Helpers should work without hash (#)

### ISSUE THREAD

**Issue by sarihac-vortexsolution** [salesforce/handlebars-php#21]
Hi,
I'd like to add custom helpers, but it seems to not support if there is no hash.
I want something like {{log post}} for example, like in the [javascript handlebars library](https://handlebarsjs.com/guide/builtin-helpers.html#log).

thanks !

**Comment by theodorejb** [comment]
Non-block helpers work correctly in [PHP Handlebars](https://github.com/devtheorem/php-handlebars), a modern project I created focused on high performance and full Handlebars spec support.
